css/style.css

```diff
@@ -0,0 +1,206 @@
+:root {
+  --baseGray-40: #b5bec6;
+  --baseWhite: #ffffff;
+  --baseGray-80: #4a5660;
+  --date-hover: #cbbcbc;
+}
+
+[data-theme="dark"] {
```

Comment:
@raaj-oo7 Why we don't have `--baseGray-40` in dark theme variable?

Reply:
--baseGray-40 is don"t change in dark theme